documentation/src/main/resources/pages/ditto/protocol-specification-things-messages.md

```diff
@@ -99,7 +99,7 @@ The response we would get from our coffee machine could look something like this
         "content-type": "text/plain",
         "version": 1,
     },
-    "path": "/inbox/messages/ask",
+    "path": "/outbox/messages/ask",
```

Comment:
```suggestion
    "path": "/inbox/messages/ask",
```

Reply:
Can I force-push or is it better to make second commit?

Reply:
Question for future PRs: am I expected to mark conversation as resolved or author of the comment?

Reply:
Either way works .. ;)